ClientSideKeyVaultKeyToCustomerManagedKey/README.md

```diff
@@ -0,0 +1,18 @@
+# KeyVault Client Side Encryption Migration to Customer Managed Server Side Encryption
+
+This program functions as a migration of data in Azure Storage from client side encryption with KeyVault to server side encryption using a Customer Managed key through encryption scope.
+
+## Getting Started
+### Prerequisites
+[add other requirements here]
```

Comment:
is this going to be filled?

Reply:
just filled this in and added a general README